pkg/block/fetcher_test.go

```diff
@@ -35,6 +35,23 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+type testTxLabeledGauge struct {
```

Comment:
Why we need this?

Reply:
It's needed to mock txGauge in the tests. It was already there as `LabeledGauge`, so I respected it. It's not much of an effort to remove it if it's the way you want to go.

Reply:
What's the benefit of mocking?

Reply:
if w can use just Gauge?

Reply:
Nothing actually. I've fixed it, I'll push it with other changes soon.